src/stackanalysismodule.ts

```diff
@@ -46,11 +46,13 @@ export module stackanalysismodule {
               let thatContext: any;
               options['uri'] = `${
                 Apiendpoint.STACK_API_URL
-              }stack-analyses/?user_key=${Apiendpoint.STACK_API_USER_KEY}`;
+                }stack-analyses/?user_key=${Apiendpoint.STACK_API_USER_KEY}`;
               options['formData'] = payloadData;
               options['headers'] = {
                 showTransitiveReport: 'true',
                 origin: 'vscode',
+                'security-vendor': vendor,
```

Comment:
I'm confused again, for component analysis `_` has been used and here it is `-`. What is the requirement from backend?